harness_tests/adc/main1.c

```diff
@@ -0,0 +1,74 @@
+#include <test/test.h>
+#include <uart/uart.h>
+#include <adc/adc.h>
+#include <adc/pay.h>
+
+pin_info_t cs = {
+    .port = &ADC_CS_PORT_PAY,
+    .ddr = &ADC_CS_DDR_PAY,
+    .pin = ADC_CS_PIN_PAY
+};
+
+adc_t adc = {
+    .channels = 0x0c00, // poll pins 10 and 11 of the ADC in auto-1 mode
+    .cs = &cs
+};
+
+//any way we can test send_adc?
```

Comment:
You don't need to since it's integrated with the other functions and checked anyways.